src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/model/Deltaker.kt

```diff
@@ -107,9 +108,9 @@ data class Deltaker(
 
             Tiltakstype.Tiltakskode.ARBEIDSRETTET_REHABILITERING,
             Tiltakstype.Tiltakskode.AVKLARING,
-            -> weeks(12)
+            -> weeks(16)
```

Comment:
Det er ikke veldig viktig, men jeg synes kanskje det hadde vært fint å definert en variabel eller noe som `ferieTillegg = 4` og hatt max være `weeks(12 + ferieTillegg)`. Bare for å gjøre det tydeligere for fremtiden.